spec/acceptance/server_instance_spec.rb

```diff
@@ -3,7 +3,7 @@
 # run a test task
 require 'spec_helper_acceptance'
 
-describe 'postgresql instance test1', if: os[:family] == 'redhat' && os[:release].start_with?('8') do
+describe 'postgresql instance test1', if: os[:family] == 'redhat' && os[:release].split.first.to_i >= 8 do
```

Comment:
I'd just exclude EL7. IMHO easier to read.
```suggestion
describe 'postgresql instance test1', if: os[:family] == 'redhat' && !os[:release].start_with?('7') do
```